src/lib/y2network/config_reader/sysconfig.rb

```diff
@@ -69,21 +74,20 @@ def find_interfaces
       # TODO: currently it implicitly loads main/default routing table
       #
       # return [RoutingTable] an object with routes
-      def load_routes
-        # load /etc/sysconfig/network/routes
-        routing_table = SysconfigRoutesReader.new.config
-        # load /etc/sysconfig/network/ifroute-*
-        dev_routing_tables = find_interfaces.map do |iface|
-          SysconfigRoutesReader.new(
-            routes_file: "/etc/sysconfig/network/ifroute-#{iface.name}"
-          ).config
+      def find_routing_tables
+        main_routes = load_routes_from
+        iface_routes = find_interfaces.flat_map do |iface|
+          load_routes_from("/etc/sysconfig/network/ifroute-#{iface.name}")
         end
+        all_routes = main_routes + iface_routes
+        [Y2Network::RoutingTable.new(all_routes.uniq)]
```

Comment:
is uniq really needed? is duplicates expected, working?

Reply:
Not sure, it was in the original code, so I just kept it.